td-agent/config.rb

```diff
@@ -9,6 +9,8 @@
 # https://www.openssl.org/source/
 OPENSSL_VERSION = "1.1.1i"
 
+BUNDLER_VERSION= "2.2.0"
+
```

Comment:
How about using the latest bundler 2.2.9?
It seems that performance regression is resolved up to 2.2.9.

Reply:
I'll do it in another PR.

Reply:
I got it.